README.rst

```diff
@@ -1,13 +1,65 @@
 This package contains the Zephyr RTOS meta tool, 'west'.
 
-To install west from this package into your user directory, run:
+Important Note
+--------------
 
-$ python3 setup.py install --user
+West is distributed in two pieces:
 
-You can also try it out by just running the package as a module from
-this directory:
+1. A bootstrap/wrapper script, which is distributed via PyPI.
+2. The "main" west package and entry points, which are fetched by the
+   bootstrap script.
+
+This somewhat unusual arrangement is because:
+
+- One of West's jobs is to manage interaction with Zephyr's multiple
+  Git repositories, including its own.
+- West is in its experimental stages and is moving quickly, meaning
+  users need to stay on HEAD.
+
+The default setup.py installs the **wrapper script only**.
+
+Installation
+------------
+
+To install the West bootstrapping/wrapper script from this package in
+development mode, clone this repository and run this from the top
+level directory:
+
+$ pip3 install -e .
+
+Then use the wrapper script to initialize a Zephyr installation with::
+
+  $ west init your/zephyr/install-dir
+  $ cd your/zephyr/install-dir
+  $ west <command>
+
+The ``west init`` call will:
+
+- create your/zephyr/install-dir
+- clone the Zephyr manifest repository (whose URL can be overridden
+  with the ``-u`` option, and branch with ``--manifest-branch``)
```

Comment:
why the inconsistency? `--manifest-branch` and `--west-revision`. Wouldn't it be better to use `--manifest-rev` (I also propose to shorten to `rev`

Reply:
Alternatively, some sort of compound URL@rev scheme to use a single param?

Reply:
Crap, that's a bug in the README. It's consistent in the code: `--manifest-rev` and `--west-rev`. They both have shorthands: `--mr` and `--wr` respectively. README now fixed.

Reply:
> Alternatively, some sort of compound URL@rev scheme to use a single param?

I think that's a bit too tricky personally. We don't know if `@` will ever be significant within a URL schema. What if I want to say `-u git+ssh://user@host/manifest`? Same could be said of any other special character.

Reply:
Let's also keep in mind that 99.9% of the time, users want master.

Reply:
Yes, I agree. I much prefer the current set of command-line options to a compound scheme. I just like to raise these on a review in case someone can think of a way of actually making it useful.